src/utils.ts

```diff
@@ -99,19 +99,16 @@ export function useThrottledValue<T = any>(value: T, ms: number = 100) {
 }
 
 export function shouldRejectKeystrokes(
-  {
-    ignoreWhenFocused,
-  }: {
-    ignoreWhenFocused: string[];
-  } = { ignoreWhenFocused: [] }
+  shortcut: string,
+  ignoreWhenFocused: string[] = [],
 ) {
   const inputs = ["input", "textarea", ...ignoreWhenFocused].map((el) =>
     el.toLowerCase()
   );
 
   const activeElement = document.activeElement;
   const ignoreStrokes =
-    activeElement &&
+    activeElement && !shortcut.startsWith('$mod') &&
```

Comment:
So the idea is if the specific shortcut short with `$mod`, we allow that shortcut's action to be called both when kbar is open & closed?

Reply:
Yep, I think in all cases when the user enters key sequence started with mod are command related

Reply:
Ah I found the problem. When kbar is opened the next combination of $mod+key closes it without action.

Reply:
@timc1 I push some updates, there are some corner cases which we need to handle.